Source/Core/DolphinQt2/Settings/GameCubePane.cpp

```diff
@@ -253,6 +252,9 @@ void GameCubePane::LoadSettings()
     case ExpansionInterface::EXIDEVICE_MEMORYCARD:
       index = 2;
       break;
+    case ExpansionInterface::EXIDEVICE_ETH:
+      index = 2;
```

Comment:
Can we use named constants, or better, an enum? Looking at this made me realise how unclear the numbers are.